hraven-etl/src/main/java/com/twitter/hraven/mapreduce/GraphiteOutputFormat.java

```diff
@@ -163,6 +167,50 @@ public void write(EnumWritable<HravenService> serviceKey, HravenRecord value) th
           throw new IOException(e);
         }  
       }
+        } else {
+            String multipleConfigInput = context.getConfiguration().get("hraven.multiple.config.files");
+            String[] multipleConfigs = multipleConfigInput.split(",");
+            for (String config : multipleConfigs) {
+                String configStr = readFsFile(config, new Configuration());
+                HashMap<String,String> configMap = new Gson().fromJson(configStr.toString(), new TypeToken<HashMap<String, String>>(){}.getType());
```

Comment:
instead of parsing string and getting conf values out of it manually, just use ObjectMapper and deseralize the entire JSON into a "GraphiteSinkConfig" POJO, with all these properties.